### PR TITLE
Adds `share link` for creating shared links.

### PR DESCRIPTION
**Pull request by bonafidehan**
Currently supports only public shared links.

PR on top of https://github.com/dropbox/dbxcli/pull/39.

### CODE REVIEW

File: cmd/share-link.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+		printLinkMetadata(sl.SharedLinkMetadata)
+	case *sharing.FolderLinkMetadata:
+		printLinkMetadata(sl.SharedLinkMetadata)
+	}
+
+	return
+}
+
+func printLinkMetadata(sl sharing.SharedLinkMetadata) {
+	fmt.Printf("%v\t%v\n", sl.Name, sl.Url)
```
Comment (diwakergupta) [inline cmd/share-link.go]: Should we make this a bit more helpful? Maybe something like 'Use this link to share the path <path>: <link>'?

File: cmd/share-link.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+)
+
+func shareLink(cmd *cobra.Command, args []string) (err error) {
+	if len(args) != 1 {
+		return errors.New("`link` requires `path`")
+	}
+
+	path := args[0]
+
+	// TODO(bonafidehan): support other visibility.
```
Comment (diwakergupta) [inline cmd/share-link.go]: Do you just want to work it into this PR?

**Comment by mbigras** [comment]
Is there any documentation about how to use this functionality? 

For example if I have some file `foo`  in the current directory, how would I generate a sharelink for `foo`?

**Comment by biounix** [comment]
To anybody interested in that option:

https://github.com/dropbox/dbxcli/issues/113#issuecomment-667919621

**Pull request closed without merging**